lib/core/src/Cardano/Wallet/TokenMetadata/MockServer.hs

```diff
@@ -93,20 +111,41 @@ import qualified Data.Text.Encoding as T
 type MetadataQueryApi = "metadata" :> "query"
     :> ReqBody '[JSON] BatchRequest :> Post '[JSON] BatchResponse
 
--- | Start a metadata server.
+-- | Start a metadata server on any random port.
```

Comment:
```suggestion
-- | Start a metadata server on any random port, and with no middleware.
```
(fully describing the difference to `withMetadataServerExt `)

Reply:
Thanks, I renamed and clarified these.